tests/pytest_tests/system_tests/test_core/test_model_workflow.py

```diff
@@ -11,3 +14,78 @@ def test_offline_link_artifact(wandb_init):
     with pytest.raises(NotImplementedError):
         run.link_artifact(FakeArtifact(), "entity/project/portfolio", "latest")
     run.finish()
+
+
+@pytest.mark.nexus_failure(feature="models")
+def test_log_model(relay_server, wandb_init):
+    with relay_server():
+        run = wandb_init()
+        with tempfile.TemporaryDirectory(dir="./") as tmpdir:
+            with open(tmpdir + "/boom.txt", "w") as f:
+                f.write("testing")
+
+            local_path = f"{tmpdir}/boom.txt"
+            run.log_model(local_path, model_name="test-model")
+        run.finish()
+
+        run = wandb_init()
+        download_path = run.use_model("test-model:v0")
+        file = download_path
+        assert file == f"{env.get_artifact_dir()}/test-model:v0/boom.txt"
+        run.finish()
+
+
+@pytest.mark.nexus_failure(feature="models")
+def test_use_model(relay_server, wandb_init):
+    with relay_server():
+        run = wandb_init()
+        with tempfile.TemporaryDirectory(dir="./") as tmpdir:
```

Comment:
why does it have to be a local temp dir? can you use the standard `tmp_path` fixture instead (https://docs.pytest.org/en/6.2.x/tmpdir.html)?